Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -8514,6 +8514,11 @@ void WebPageProxy::setMuted(WebCore::MediaProducerMutedStateFlags state, FromApp
     if (!isAllowedToChangeMuteState())
         state.add(WebCore::MediaProducer::MediaStreamCaptureIsMuted);
 
+#if ENABLE(MEDIA_STREAM) && ENABLE(GPU_PROCESS)
+    if (preferences().captureAudioInGPUProcessEnabled() && internals().mutedState.contains(WebCore::MediaProducerMutedState::AudioCaptureIsMuted) && !state.contains(WebCore::MediaProducerMutedState::AudioCaptureIsMuted))
```

Comment:
Should we make it impossible to enable `UseMicrophoneMuteStatusAPI` when `captureAudioInGPUProcessEnabled` is false?

Reply:
Actually, it would be better to allow it.
I'll work on enabling this in a follow-up, this patch is already big enough.

Reply:
I filed https://bugs.webkit.org/show_bug.cgi?id=280235